package/src/custom/clients/base/fields/Ratingfield/Ratingfield.js

```diff
@@ -0,0 +1,35 @@
+({
+    /**
```

Comment:
We should probably have a docblock that explains what the RatingField is. Maybe have a link back to ApplicationRatings.md doc.